replica/replica.go

```diff
@@ -563,9 +563,12 @@ func (r *Replica) PrepareRemoveDisk(name string) ([]PrepareRemoveAction, error)
 		disk = GenerateSnapshotDiskName(name)
 		data, exists = r.diskData[disk]
 		if !exists {
-			return nil, fmt.Errorf("Can not find snapshot %v", disk)
+			return nil, nil
 		}
 	}
+	if data.Removed {
```

Comment:
so, if the disk is already in removed state, then, it never gets deleted.
case like where disk is marked as removed as part of deletion process, and replica got disconnected.

Reply:
removed this check, thanks.